src/general/Jukebox.cs

```diff
@@ -538,6 +543,8 @@ private void OnCategoryEnded()
             {
                 foreach (var track in list.Tracks)
                 {
+                    track.PlayedOnce = false;
```

Comment:
I found a bug regarding this (this reset needs to *always* happen when `category` is not null.
If you go into a new game and go extinct, the track plays fine once, but if you exit to menu and then start a new game where you go extinct again, you don't hear the extinct track again even though you should.